index.bs

```diff
@@ -2392,6 +2392,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain pick rate for a source with attribution scope</dfn> given a positive integer |numTriggerStates|, a positive integer |numEventStates|, a non-negative 32-bit integer |numUsedScopes|, and a double |epsilon|:
```

Comment:
This method name seems a bit confusing since we always use it but generally we define two types of sources, with and without attribution scope.